src/main/haskell/kore/src/Kore/Step/ExpandedPattern.hs

```diff
@@ -216,9 +217,20 @@ isBottom
   = True
 isBottom _ = False
 
+{-| If an ExpandedPattern is bottom, it is converted to the canonical
+representation of bottom.
+-}
+loBottomize
```

Comment:
Are we now requiring to normalized `\bottom` `ExpandedPattern`s? If so, should we combine this with other normalizers for `ExpandedPattern`? If not, why is it required?

I also think the joke is going to be lost on people.